README.md

```diff
@@ -16,13 +19,27 @@ And then execute:
 
     $ bundle install
 
-Or install it yourself as:
+## Usage
 
-    $ gem install lightweight_serializer
+To create a serializer, create a new file in your `app/serializers/` folder. Let's assume we have a nice blog:
 
-## Usage
+```ruby
+class PostSerializer < LightweightSerializer::Serializer
+  attribute :title
+  attribute :content
 
-TODO: Write usage instructions here
+  collection :comments, serializer: CommentSerializer
+
+  nested :author, serializer: UserSerializer
```

Comment:
This example doesn't really explain what the difference between `nested` and `collection` is.

Also, an output example would be nice.

Reply:
This is WIP

Reply:
Oh, I thought opening that PR means that you're open for comments (this is the process that I know from other projects, but it might differ here, all good) and the comment was nice enough. Anyway, I'll wait for the request then. Have a great Friday :)